package.json

```diff
@@ -27,12 +30,13 @@
 		"build-clean": "rimraf ./packages/*/build ./packages/*/build-module",
 		"prebuild": "check-node-version --package",
 		"build": "node ./scripts/build.js",
-		"postinstall": "lerna bootstrap --hoist && ./scripts/create-symlinks.sh && npm run build",
+		"create-symlinks": "node ./scripts/create-symlinks.js",
+		"postinstall": "lerna bootstrap --hoist && npm-run-all create-symlinks build",
 		"test": "wp-scripts test",
 		"test:coverage": "npm run test -- --coverage",
 		"test:coverage-ci": "npm run test -- --coverage && codecov",
 		"test:watch": "npm run test -- --watch",
-		"publish:dev": "npm run build-clean && npm run build && lerna publish --npm-tag next",
-		"publish:prod": "npm run build-clean && npm run build && lerna publish"
+		"publish:dev": "npm-run-all build-clean build && lerna publish --npm-tag next",
```

Comment:
I know it's unrelated, but I felt tempted to refactor as I added `npm-run-all` 😃

Reply:
If `build-clean` is a preparation step for build, why not include in `prebuild` ?